Sources/SolutionTester/SolutionTester.swift

```diff
@@ -86,19 +83,3 @@ public class SolutionTester {
 
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/eneko/Ninety-Nine-Swift-Solutions/blob/6cf3be9db0e01390503f2acc40ecd9b2665bfa73/Sources/SolutionTester/SolutionTester.swift#L85"></span>Files should have a single trailing newline.</td>
    </tr>
  </tbody>
</table>